aws/sdk/build.gradle.kts

```diff
@@ -166,7 +169,13 @@ fun discoverServices(): List<AwsService> {
                 } else {
                     listOf()
                 }
-                AwsService(service = service.id.toString(), module = sdkId, modelFile = file, extraFiles = extras)
+                AwsService(
+                    service = service.id.toString(),
+                    module = sdkId,
+                    moduleDescription = "AWS SDK for $title",
```

Comment:
Examples for this value:
- AWS SDK for Amazon Simple Storage Service
- AWS SDK for AWS Identity and Access Management
- AWS SDK for Amazon DynamoDB

I think I would prefer to it to be "AWS SDK for S3", but I'm not sure if there's a reliable model attribute for that. One would think the Service `sdkId` would work, but that value doesn't look great for a number of services.